indexers/vector/SptagIndexer/tests/test_sptagindexer.py

```diff
@@ -75,18 +75,18 @@ def test_sptag_indexer_known(metas):
     with BaseIndexer.load(save_abspath) as indexer:
         assert isinstance(indexer, SptagIndexer)
         idx, dist = indexer.query(queries, top_k=2)
-        np.testing.assert_equal(idx, np.array([[4, 5], [5, 4], [6, 5], [7, 6]]))
+        np.testing.assert_equal(idx, np.array([[4., 5.], [5., 4.], [6., 5.], [7., 6.]]))
```

Comment:
this seems weird to me, it should be equal to the `np.array` as type str

Reply:
the `_int2ext_id` method is implemented in `BaseNumpyIndexer` and not `BaseIndexer`